src/content/guides/limiting-pod-communication-with-network-policies/index.md

```diff
@@ -1,7 +1,7 @@
 +++
 title = "Limiting Pod Communication with Network Policies"
 description = "Guide on how to limit Pod communication using Network Policies"
-date = "2017-11-10"
+date = "2019-11-12"
```

Comment:
```suggestion
date = "2019-11-07"
```

A date in the future will make this page invisible until we build again on or after that date.